renderer/components/Request/RequestSummary.js

```diff
@@ -114,10 +160,10 @@ const RequestSummary = ({ invoice = {}, payReq, intl, showNotification, ...rest
         }
         right={
           <Text>
-            {payReq ? (
-              <QRCode value={payReq.toUpperCase()} />
-            ) : (
+            {isKeysend ? (
               <FormattedMessage {...messages.payment_request_keysend} />
+            ) : (
+              <QRCode value={paymentRequest.toUpperCase()} />
```

Comment:
Do we need to check for `paymentRequest` to be != null ? Looks like it's possible (isKeySend being false doesn't automatically mean that paymentRequest is set or does it?).